browser-extensions/common/js/lib/i18n.js

```diff
@@ -419,7 +419,7 @@ var domains = {
             "コース状況確認係": "Event Day Course Check"
         }
     },
-    "parkrun.co.nl": {
+    "www.parkrun.co.nl": {
```

Comment:
This must have been broken for a while? I'm sure it must have worked once though... That's the problem with no tests, I've no idea!

Reply:
Yes, I concluded it probably had been for a while.